build-system/pr-check.js

```diff
@@ -224,6 +224,7 @@ const command = {
     timedExecOrDie(`${gulp} clean`);
   },
   runLintChecks: function() {
+    timedExecOrDie(`${gulp} json-syntax`);
```

Comment:
Depending on how long `gulp json-syntax` is expected to take, it could be run before or after `gulp lint`, which takes about 1m 20s to run. It makes a difference on Travis if we early-exit by running quicker tests before longer-running ones.